lms/models/__init__.py

```diff
@@ -1,11 +1,13 @@
 from lms.models.application_instance import ApplicationInstance
+from lms.models.lis_result_sourcedid import LISResultSourcedId
```

Comment:
Note change in casing here. Proof of concept had `LISResultSourceDID`. In LIS/LTI outcome docs, the field is referred to as `sourcedId` so I wanted to not capitalize the `d` in `Sourced`, and was trying to match casing of other models a little more closely here. No two ways about it, this term is hell for casing.